escrow/free_call_state_service_test.go

```diff
@@ -0,0 +1,115 @@
+
+	suite.orgMetaData, _ = blockchain.InitOrganizationMetaDataFromJson(testJsonOrgGroupData)
+	suite.serviceMetaData, _ = blockchain.InitServiceMetaDataFromJson(testJsonData)
+	suite.memoryStorage = NewMemStorage()
+	suite.storage = NewFreeCallUserStorage(suite.memoryStorage)
+	suite.service = NewFreeCallUserService(suite.storage,
+		NewEtcdLocker(suite.memoryStorage, suite.serviceMetaData), func() ([32]byte, error) { return suite.orgMetaData.GetGroupId(), nil },
+		suite.serviceMetaData)
+	suite.stateService = NewFreeCallStateService(suite.orgMetaData, suite.serviceMetaData, suite.service, suite.freeCallPaymentValidator)
+
+
+}
+
+func (suite *FreeCallStateServiceSuite) TestGetFreeCallsAvailable() {
```

Comment:
Does this also test the decrease in the number of free calls post service invocation?

Reply:
this test case is for state service , for increase in the free call count, we have a seprate test case